pallets/schemas/src/lib.rs

```diff
@@ -122,24 +122,9 @@ pub mod pallet {
 		/// The schema model exceeds the maximum length allowed
 		ExceedsMaxSchemaModelBytes,
 
-		/// The governance schema model max value provided is too large (greater than the BoundedVec size)
-		ExceedsGovernanceSchemaModelMaxValue,
-
 		/// The schema is less than the minimum length allowed
 		LessThanMinSchemaModelBytes,
 
-		/// Schema does not exist
-		NoSuchSchema,
```

Comment:
It would be nice to know what happened to these errors; were they ever in use?

Reply:
I wonder if this particular error was replaced by `InvalidSchemaId` in the messages pallet:
```
let schema = T::SchemaProvider::get_schema_by_id(schema_id);
			ensure!(schema.is_some(), Error::<T>::InvalidSchemaId);
```